src/core/components/Paginate/Paginate.scss

```diff
@@ -25,13 +29,29 @@
 }
 
 .Paginator-previous {
-  &::before {
-    @include arrow-previous();
+  [dir=lrt] & {
+    &::before {
```

Comment:
Hah, right. I was doing something different originally but that makes sense now that I did it this way 👍